unit-tests/resources/commcare-apps/case_search_and_claim/default/app_strings.txt

```diff
@@ -0,0 +1,45 @@
+app.display.name=village-health2
```

Comment:
ah nevermind, I forgot it gets referenced in the suite file. looks like all the other test apps have this file as well.